node/lib/vsotask.ts

```diff
@@ -317,6 +337,14 @@ export function find(findPath: string): string[] {
 export function rmRF(path: string): void {
     debug('rm -rf ' + path);
     shell.rm('-rf', path);
+    
+    var error: string = shell.error();
+    // if you try to delete a file that doesn't exist, desired result is achieved
+    // other errors are valid
+    if (error && !(error.indexOf('ENOENT') === 0)) {        
+        console.error(error)
+        exit(1);
```

Comment:
I'm not sure we want exit code 1 communicated to the user.  That's fine if you're executing a tool.  

I think this needs to call setResult Fail with exit arg.  BTW, I think rmRF can take an optional arg continueOnError (which would be falsey if not suppled and exit).  But it gives task authors the ability to make clean up a best effort.